guides/common/modules/proc_upgrading-project-in-place-using-leapp.adoc

```diff
@@ -43,13 +52,64 @@ On {RHEL}, enable the `rhel-7-server-extras-rpms` repository:
 ----
 # subscription-manager repos --enable=rhel-7-server-extras-rpms
 ----
+ifdef::satellite[]
+.. Enable the `{RepoRHEL7Server}` repository:
```

Comment:
I am unsure if you want to merge anything about rhel7 to master. cc @evgeni 


It's probably el8 only by now according to our milestone. And el8 has IMHO no "server" variant.

Reply:
this whole file doesn't really apply to master anymore, but we've kept it here for easier making changes and picking.

the *content* is correct.